.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
+  'env': {
+    'browser': true,
+    'node': true
+  },
+  'globals': {
+    'window': true,
+    'document': true,
+    'jQuery': true,
+    '$': true,
+    'angular': true
```

Comment:
Unfortunately, I don't think that react will be a good solution for this due to the current requirement of slim. The html tags are essentially modified by regular expressions (I think) to include angular directives in the editor. 
That being said, we should get away from using jquery, but I am currently crippled my my lack of understanding of angular. We might have a little better context once @samcdavid gets back from vacation.

All that to say, yes, this could be better, and hopefully it soon will be.